emackinnon1/refactor-tractor
# Refactor scripts

#### What's this PR do?
Refactor scripts file, clean up comments
Add .then methods to Promise.all
Write functions to display data on main display
Rewrite innerText query selectors to use jQuery
Run Prettier on all files

#### Where should the reviewer start (files changed)?
scripts.js

#### How should this be manually tested?
We will need to use spies to test DOM methods
One test on classes not passing (see screenshot)

#### Any background context you want to provide?
n/a

#### What are the relevant tickets?
n/a

#### Screenshots (if appropriate)
<img width="504" alt="Screen Shot 2020-04-09 at 8 42 10 PM" src="https://user-images.githubusercontent.com/54958511/78957690-4854db00-7aa3-11ea-894b-e8f278efd651.png">

#### Questions:
When we add our form to post data do we want it displayed on main display or should we make it a dropdown?
